pkg/provider/providerGetter.go

```diff
@@ -15,7 +16,7 @@ import (
 var knownProviders = []k8sv1.Provider{k8sv1.ProviderVelero}
 
 var NewVeleroProvider = func(k8sClient K8sClient, ecosystemClientSet EcosystemClientSet, recorder EventRecorder, namespace string) Provider {
```

Comment:
```suggestion
var NewVeleroProvider = func(k8sClient K8sClient, recorder EventRecorder, namespace string) Provider {
```

ecosystemClientSet EcosystemClientSet required?